test/make.sh

```diff
@@ -18,8 +18,7 @@
 # shebang has a '- e' flag, which causes it
 # to exit on error
 function check_bash() {
-find . -name "*.sh" -not -path "*/istio-1.0.1/*" \
-  -not -path "*/gke-istio-shared/*" | while IFS= read -d '' -r file;
+find . -name "*.sh" | while IFS= read -d '' -r file;
```

Comment:
We need to ignore the gke-istio-shared files. These should be checked in that repo. once they get here, we should think of them as a black box. It's just a third party library

Reply:
We need to update this in the main repo, not here